src/DSFML/Audio/SoundBuffer.cpp

```diff
@@ -68,17 +68,17 @@ void sfSoundBuffer_destroy(sfSoundBuffer* soundBuffer)
     delete soundBuffer;
 }
 
-DBool sfSoundBuffer_saveToFile(const sfSoundBuffer* soundBuffer, const char* filename)
+DBool sfSoundBuffer_saveToFile(const sfSoundBuffer* soundBuffer, const char* filename, size_t length)
 {
-    return (soundBuffer->This.saveToFile(filename))? DTrue: DFalse;
+    return (soundBuffer->This.saveToFile(std::string(filename, length)))? DTrue: DFalse;
 }
 
 const DShort* sfSoundBuffer_getSamples(const sfSoundBuffer* soundBuffer)
 {
     return soundBuffer->This.getSamples();
 }
 
-size_t sfSoundBuffer_getSampleCount(const sfSoundBuffer* soundBuffer)
```

Comment:
I made the change here because sf::SoundBuffer::getSampleCount returns a Uint64, not size_t; I forgot to make the change D-side though! I'll fix that now.

c.f. http://www.sfml-dev.org/documentation/2.3.2/classsf_1_1SoundBuffer.php#a2a791e7304553fa96269cc355cc4f7e8